tools/changelogger-release.sh

```diff
@@ -171,6 +171,19 @@ for SLUG in "${TO_RELEASE[@]}"; do
 	info "Processing $SLUG..."
 	RELEASED[$SLUG]=1
 
+	# Avoid "There are no changes with content for this write. Proceed?" prompts and empty changelog entries.
+	ANY=false
+	for f in "$CHANGES_DIR"/*; do
+		if [[ -n $( sed -n '/^$/,$ { /[^ \t]/ p }' "$CHANGES_DIR"/* ) ]]; then
```

Comment:
```suggestion
		if [[ -n $( sed -n -e '/^$/,$ { /[^ \t]/ p }' "$CHANGES_DIR"/* ) ]]; then
```
Without the `-e`, this sed outputs the following:

![Screenshot 2023-11-29 at 12 50 58 PM](https://github.com/Automattic/jetpack/assets/33553323/3fc2a1ee-8f1a-4863-b644-5fa93742ec92)

Which seems to be from MacOS using a different version of sed as you indicated.

Also is it supposed to be:

```suggestion
		if [[ -n $( sed -n -e '/^$/,$ { /[^ \t]/ p }' "$f" ) ]]; then
```
?

Reply:
Yes, good catches on both!